test/test-feel.js

```diff
@@ -180,26 +163,45 @@ for (const file of fs.readdirSync(caseDir)) {
 
       it(name, () => run(createParser(context)));
     }
+
+
+    describe('with custom variable context', function() {
+
+      const contextTracker = (context) => {
+        return trackVariables(toEntriesContextValue(context), EntriesContext);
+      };
+
+      for (const { name: testName, run } of tests) {
+
+        const {
+          it,
+          name,
+          context
+        } = parseTest(testName, contextTracker);
+
+        it(name, () => run(createParser(context)));
```

Comment:
To use the custom Context tracker when executing the test, it should be:

```suggestion
        } = parseTest(testName);

        it(name, () => run(createParser(context), contextTracker));
```

Even with the fix, it still uses the default `VariableContext` sometimes. I was not able to find the root-cause just yet. I will look into it further

Reply:
Can we create a test case for this, as close to your actual usage scenario?

Reply:
Fixed via https://github.com/nikku/lezer-feel/pull/18/commits/eafc3e1bccb2545b2bc55bea0dae7fce9f377692.

Reply:
> Can we create a test case for this, as close to your actual usage scenario?

Not sure if I have time for it today, but I will look into it